### PR TITLE
fix mmdeploy.version_info

### PR DESCRIPTION
**Pull request by xiabo123**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily receiving feedbacks. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

fix mmdeploy.version_info
image

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repositories?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit tests to ensure the correctness.
3. If the modification has a dependency on downstream projects of a newer version, this PR should be tested with all supported versions of downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: mmdeploy/__init__.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 import importlib
 
 from mmdeploy.utils import get_root_logger
-from .version import __version__  # noqa F401
+from .version import *  # noqa F401
```
Comment (RunningLeon) [inline mmdeploy/__init__.py:5]: hi, could you elaborate more on the motivation you use `version_info` instead of `mmdeploy.__version__`.

Comment (RunningLeon) [inline mmdeploy/__init__.py:5]: could only import `__version__` and `version_info`

```

from .version import __version__, version_info # noqa F401
```

Comment (xiabo123) [reply mmdeploy/__init__.py:5]: in that case, in 'version. py' “__all__ = ['__version__', 'version_info', 'parse_version_info']”,What is the meaning of it?
Function "parse_version_info", The same error occurs with info.

Comment (RunningLeon) [reply mmdeploy/__init__.py:5]: then, we could remove `parse_version_info` from `__all__`

Comment (xiabo123) [reply mmdeploy/__init__.py:5]: Thank you for your answer. If you don't use the feature of '__all__' in this way, this PR can be modified as you mentioned. Do you think it's okay?
`from .version import __version__, version_info # noqa F401`

Comment (xiabo123) [reply mmdeploy/__init__.py:5]: Previously, the training referred to the "mmcv" library and compared versions using this. I encountered this problem when using 'mmdeploy' for reasoning.

Comment (RunningLeon) [reply mmdeploy/__init__.py:5]: works for me.

Comment (RunningLeon) [reply mmdeploy/__init__.py:5]: @xiabo123 hi, when will you update this PR?

Comment (xiabo123) [reply mmdeploy/__init__.py]: Has been updated

**Comment by RunningLeon** [comment]
@xiabo123 could you fix the lint error by running `pip install pre-commit && pre-commit install && pre-commit run --all-files`.

**Comment by xiabo123** [comment]
ok

**Comment by xiabo123** [comment]
@RunningLeon Have you encountered the following problems? During Flake8, the version number of mccabe was always incorrect. Both 0.6.0 and 0.6.1 cannot be installed in practice.
3b7e5231c69f4a9c62821ad671142cb

**Comment by xiabo123** [comment]
@RunningLeon 
`from .version import __version__, version_info # noqa F401`
I'm very sorry, the modification 'version_info' was not called in 'mmdeploy' and cannot be checked by 'isort'.
I attempted to make modifications in our call layer.
`from mmdeploy.version import version_info`
We apologize for any inconvenience caused.

**Comment by RunningLeon** [comment]
@xiabo123 maybe you could use here to parse package version. Cause not all packages provide `version_info`
```
import mmdeploy
from packaging import version # pip install packaging if not installed
v = version.parse(mmdeploy.__version__)
print(v.major, v.minor, v.micro)
```

**Comment by xiabo123** [comment]
@RunningLeon OK,thank you.

**Review by RunningLeon — COMMENTED** [review]
LGTM

**Pull request merged**